Mail/Views/Menu Drawer/Items/MenuDrawerItemsListView.swift

```diff
@@ -27,6 +27,8 @@ import SwiftUI
 struct MenuDrawerItemsAdvancedListView: View {
     @State private var isShowingRestoreMails = false
 
+    @LazyInjectService var urlNavigator: URLNavigable
```

Comment:
Sorry I didn't notice earlier but in SwiftUI we have this nice environment value that we can use everywhere:
`@Environment(\.openURL) var openURL` this does the same as URLNavigable but it's less code for us to maintain.

Reply:
Sure. After looking at it, it does look like everything that opens an URL is SwiftUI code, so I can use it. Bottomline I need to abstract UIApplication.

Reply:
done